src/stratigraphy/depthcolumn/depthcolumn.py

```diff
@@ -83,11 +98,21 @@ class DepthColumnFactory:
 
     @staticmethod
     def create(data: dict) -> DepthColumn:
+        """Creates a DepthColumn object from a dictionary.
+
+        Args:
+            data (dict): A dictionary representing the depth column.
+
+        Returns:
+            DepthColumn: The depth column object.
+        """
         column_type = data.get("type")
         if column_type == "BoundaryDepthColumn":
             return BoundaryDepthColumn.from_json(data)
         elif column_type == "LayerDepthColumn":
             return LayerDepthColumn.from_json(data)
+        elif column_type == "LayerIdentifierColumn":
+            return LayerIdentifierColumn.from_json(data)
```

Comment:
I think it would be better. However, I thought the refactoring might be out of scope here, and I did not know whether it would make sense. If i does make sense, I can refactor that class with you, or we can open a follow-up ticket that aims to clean these classes and their inheritance patterns.